demos/node-classification/gcn/gcn-cora-node-classification-example.ipynb

```diff
@@ -4,14 +4,36 @@
    "cell_type": "markdown",
```

Comment:
<p>You might want to emphasize here that StellarGraph models are built using custom Keras layers and as such, we can use them together with standard Keras layers. For example, here is how to use a Keras Dense layer for classification using the node embeddings from the GCN layers.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>